en/lessons/basics/operators.md

```diff
@@ -0,0 +1,185 @@
+---
+version: 1.0.0
+title: Operators
+---
+
+Haskell provides constructs for using infix binary functions, called operators. In
+this lesson you will learn how to define operators and control the parsing of
+operators, and how infix and prefix syntax can be interchanged.
+
+{% include toc.html %}
+
+
+## Using Operators
+
+Operators in Haskell are functions that can be placed between two arguments: they can
+return a value of any type and their names are either a string of symbols or ordinary
+function names surrounded in backticks. Using infix notation makes it easier to write
+expressions containing many common functions:
+
+```console?lang=haskell&prompt=ghci>,ghci|
+ghci> abs . subtract 119 $ 2 + 2
+115
+ghci> (`mod` 930233356) . negate $ 4
+930233352
+ghci> - (7 + 57843)
+-57850
+```
+__Note__: negate (`-`) is the only unary operator in Haskell.
```

Comment:
this might be a bit confusing since the `-` is overloaded and can be used for subtraction too.

Reply:
Furthermore, my note doesn't say what use knowing that "-" is also a unary operator is: I may give an example of how it makes sectioning more difficult.